### PR TITLE
Use longer G4 range in calo volume

### PR DESCRIPTION
**Pull request by brownd1978**

### CODE REVIEW

**Comment by kutschke** [comment]
@FNALbuild run build tests

**Review by kutschke — APPROVED** [review]
I resubmitted the tests after merging PR 331.  I full expect the tests to pass again and will merge after they do.

File: JobConfig/primary/epilog.fcl
Comment (resnegfk) [inline JobConfig/primary/epilog.fcl:3]: This comment should be changed as it is misleading. It is production range cut for that region

**Pull request merged**
